independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/core/CodestartProjectGeneration.java

```diff
@@ -34,10 +34,10 @@ static void generateProject(final CodestartProjectDefinition projectDefinition,
 
         log.debug("processed shared-data: %s" + data);
 
-        final Codestart projectCodestart = projectDefinition.getRequiredCodestart(CodestartType.PROJECT);
-
         final List<CodestartFileStrategy> strategies = buildStrategies(mergeStrategies(projectDefinition));
 
+        projectDefinition.getRequiredCodestart(CodestartType.PROJECT);
```

Comment:
Is there a reason why you moved the line? I think we should keep it at the place it was before.

Reply:
That's a mistake, when reverting this, I pasted the line at the wrong place. I moved it back to its original place.